src/index.ts

```diff
@@ -18,11 +18,9 @@ import { LogTimings } from './flow-control/log-timings';
 import { RestartProcess } from './flow-control/restart-process';
 import { Logger } from './logger';
 
+/** Logger options  */
```

Comment:
I didn't go through the .md file yet, but it's my perception (and, well, it's what we do at Canva too) that `//` comments are usually not picked up by tooling (doc generators and IDEs), whereas `/* */` comments are, so they work mostly as a file-internal comment that doesn't need to be exposed.

The old `// Logger options` was here mostly to create a section (just like there are others for input handling, restarting and so on below), I don't think it adds any value being in `/* */` nor is it correct as the doc for `ConcurrentlyOptions`

cc @paescuj just because I think this has been an unwritten rule in this codebase :)

Reply:
I modified theses comments for `ts-readme` 

they should be reverted after migrating to `typedoc` : (cf: https://github.com/open-cli-tools/concurrently/pull/445#discussion_r1343516868)

Reply:
Reverted in 982c15fcfb1e747742f0e45a5fbf14b1028ccdac